shotover-proxy/src/server.rs

```diff
@@ -459,6 +469,45 @@ impl<C: Codec + 'static> Handler<C> {
 
             debug!("Received raw message {:?}", messages);
 
+            {
+                let mut already_modified = false;
+
+                for i in 0..messages.len() {
+                    if let Frame::Cassandra(cf) = &messages[i].original {
```

Comment:
I'm going to modify this to only parse the Opcode, and only do a full parse if the query is a `USE` statement

Reply:
You wont be able to tell if its a USE just from the opcode...?

Reply:
True, we will have to check the opcode is a Query, and then parse the first bit of the CQL to determine that.

Reply:
Honestly I would just parse it the normal way at that point.

Reply:
Maybe we need some kind of toggle for this feature or detect if the transforms used need this set.

Reply:
Yeah, we did discuss that. A flag for if a transform is using the `Message::namespace` method is probably the way to go about it.